predicators/envs/repeated_nextto.py

```diff
@@ -26,7 +26,7 @@ class RepeatedNextToEnv(BaseEnv):
     env_lb: ClassVar[float] = 0.0
     env_ub: ClassVar[float] = 100.0
     grasped_thresh: ClassVar[float] = 0.5
-    nextto_thresh: ClassVar[float] = 0.5
+    nextto_thresh: ClassVar[float] = CFG.repeated_nextto_nextto_thresh
```

Comment:
we can run into trouble sometimes if we use `CFG` in defining a class variable, I believe particularly in testing. so instead, can you initialize this field inside of `__init__`?

Reply:
is `__init__` allowed to take in different arguments than just `self` and `use_gui`?

Reply:
I just meant like this: https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/predicators/envs/blocks.py#L94